tactics/redexpr.mli

```diff
@@ -54,6 +54,10 @@ val set_strategy :
 (** call by value normalisation function using the virtual machine *)
 val cbv_vm : reduction_function
 
+(** [subst_red_expr sub c] performs the substitution [sub] on all kernel
+   names appearing in [c] *)
+val subst_red_expr : Mod_subst.substitution -> red_expr -> red_expr
```

Comment:
This is not strictly needed (I thought it was used, but instead we need `Tacsubst.subst_redexp`, but there is no harm in exposing it, right?

Reply:
OK to me.